app/helpers/link_helper.rb

```diff
@@ -16,7 +16,7 @@ def govuk_styled_link(text, path: nil, inverse: false)
     "<a href='#{path}' class='#{classes}'>".html_safe + text + "</a>".html_safe
   end
```

Comment:
Isn't this last commit now covered in https://github.com/alphagov/frontend/pull/4660/files ?

Reply:
Yes, I added it to get the tests to pass, but will remove it and rebase.